apps/dcos_net/src/dcos_net_mesos_listener.erl

```diff
@@ -142,6 +153,22 @@ terminate(_Reason, _State) ->
 code_change(_OldVsn, State, _Extra) ->
     {ok, State}.
 
+%%%===================================================================
+%%% Internal functions
+%%%===================================================================
+
+reset_state(State) ->
+    case State#state.ref of
```

Comment:
May I kindly ask you to not use `Foo#bar.baz` in dcos-net code? It's not readable. We prefer explicit matches, i.e. `#bar{baz = Baz} = Foo`.